test/powershell/Language/Classes/Scripting.Classes.Attributes.Tests.ps1

```diff
@@ -292,7 +292,12 @@ Describe 'ValidateSet support a dynamically generated set' -Tag "CI" {
 
             $testAssemply = "$TestDrive\tst-$(New-Guid).dll"
             Add-Type -TypeDefinition $a -OutputAssembly $testAssemply
-            Import-Module $testAssemply
+            $testModule = Import-Module $testAssemply -PassThru
+        }
+
+        AfterAll {
+            Remove-Module -ModuleInfo $testModule
+            Remove-Item $testAssemply -Force -ErrorAction SilentlyContinue

```

Comment:
We already got the `Remove-Item` error and it's not related to the `Remove-Item` line added in this PR. It's because we create the test assembly in `$TestDrive`. Pester removes `$TestDrive` after a test run, and resulted in the error.

How about we don't create the test assembly, but instead do this:
```
$cls = Add-Type -TypeDefinition $a -PassThru | select -first 1
$testModule = Import-Module $cls.Assembly -PassThru
```
I didn't try it myself but think it should work.